teammapper-backend/src/map/controllers/maps.gateway.ts

```diff
@@ -167,6 +168,26 @@ export class MapsGateway implements OnGatewayDisconnect {
     return true
   }
 
+  @UseGuards(EditGuard)
+  @SubscribeMessage('undoRedoMapChanges')
+  async undoRedoMapChanges(
```

Comment:
I would suggest to call it differently. the first time I actually struggled to understand what "undoRedo" should actually mean. e.g.: "applyMapDiff" or something like that.